src/object/srv_obj.c

```diff
@@ -2003,18 +2003,14 @@ obj_local_rw(crt_rpc_t *rpc, struct obj_io_context *ioc, struct dtx_handle *dth)
 static int
 obj_capa_check(struct ds_cont_hdl *coh, bool is_write, bool is_agg_migrate)
 {
-	if (!is_write && !ds_sec_cont_can_read_data(coh->sch_sec_capas)) {
-		if (uuid_compare(coh->sch_cont->sc_pool->spc_pool->sp_srv_cont_hdl,
-				 coh->sch_uuid) == 0)
-			return 0;
-
+	if (!is_agg_migrate && !is_write && !ds_sec_cont_can_read_data(coh->sch_sec_capas)) {
```

Comment:
probably 
if (is_agg_migrate)
        return true;

Reply:
+1

Reply:
ok, can change it in a following patch, let's save CI test time and fix the testing issue first. thanks